pkg/config/config.go

```diff
@@ -91,6 +91,10 @@ func (c *Config) SetByEnvironment(ctx *cli.Context, key string, value string) er
 	return nil
 }
 
+func getKeyByEnvironmentPath(env, path string) string {
+	return KeyEnvironment + "." + env + "." + path
```

Comment:
If more than 3 components I prefer to use `fmt.Sprintf` but it is really a personal preference.